esphome/components/ld2410s/LD2410S.cpp

```diff
@@ -0,0 +1,947 @@
+#include "LD2410S.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace ld2410s {
+
+void LD2410S::setup() {
+  this->minimal_output_ = true;
+  this->schedule_cmd_frame_(CONFIG_MODE_START_CMD);
+  this->schedule_cmd_frame_(OUTPUT_MODE_SWITCH_CMD);
+  this->schedule_cmd_frame_(FW_READ_CMD);
+  this->schedule_cmd_frame_(PARAMS_READ_CMD);
+  this->schedule_cmd_frame_(GATE_TRIGGER_THRESHOLD_READ_CMD);
+  this->schedule_cmd_frame_(GATE_HOLD_THRESHOLD_READ_CMD);
+  this->schedule_cmd_frame_(GATE_SNR_READ_CMD);
+  this->schedule_cmd_frame_(CONFIG_MODE_END_CMD);
+}
+void LD2410S::loop() {
+  if (!this->cmd_active_) {
+    if (available()) {
```

Comment:
https://developers.esphome.io/contributing/code/#c-code-style
```suggestion
    if (this->available()) {
```
Please double-check that all member variable accesses & calls are prefixed with `this->`.